packages/core/src/dynamoDb/utils/dynamodb.ts

```diff
@@ -0,0 +1,80 @@
+/*!
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { DynamoDB } from 'aws-sdk'
+import { AttributeValue, Key, ScanInput } from 'aws-sdk/clients/dynamodb'
+import { DynamoDbClient } from '../../shared/clients/dynamoDbClient'
+
+export interface RowData {
+    [key: string]: string
+}
+export interface TableData {
+    tableHeader: RowData[]
+    tableContent: RowData[]
+    lastEvaluatedKey?: Key
+}
+
+export async function getTableContent(
+    tableRequest: ScanInput,
+    regionCode: string,
+    client = new DynamoDbClient(regionCode)
+) {
+    const response = await client.scanTable(tableRequest)
+    const tableColumnsNames = getTableColumnsNames(response)
```

Comment:
If you did the below suggestion you would be able to assign values to variables in one line like:
```
const { columns, header } = getTableColumnsNames(response)
```

Reply:
Updated